whylog/config/parsers.py

```diff
@@ -62,6 +65,19 @@ def serialize(self):
             "line_content": self.line_content
         }
 
+    def convert_params(self, params):
+        converted_params = []
+        for i in range(len(params)):
+            group_type = self.convertions.get(i + 1, STRING)
+            if group_type != STRING:
+                converter = CONVERTION_MAPPING.get(group_type)
+                if converter is None:
+                    raise UnsupportedConverter(group_type)
```

Comment:
Where this exception should be catched? I have'nt found any "except" statement which would catch it.